### ISSUE TITLE
这些iou loss  代码计算完直接使用？？？

### ISSUE THREAD

**Issue by henbucuoshanghai** [Zzh-tju/DIoU-pytorch-detectron#11]
diou，ciou这些    计算完直接当loss使用？？不需要约束在负一和1之间？
为负的怎么办？？值超过1怎么办？

**Comment by Zzh-tju** [comment]
`loss = 1 - CIoU`

**Comment by henbucuoshanghai** [comment]
明白   需要1减去        但是最终的loss一定在正负1之间？哪里有约束？

**Comment by Zzh-tju** [comment]
你再看清楚一点，loss怎么会介于±1之间

**Comment by henbucuoshanghai** [comment]
那
